src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url('./fonts/Avenir-Heavy.ttf');
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, 'Helvetica', sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  width: 100%;
```

Comment:
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/70009205/200614549-5b1fb06e-c202-4196-a464-c93aefabc273.png)

Reply:
Not fixed